### DISCUSSION TITLE
How to override and change template

### DISCUSSION THREAD

**Discussion by PatelPreyansh30** [django-import-export/django-import-export#1941] [General]
How to override and change some html in the import template?

I have tried app>templates>import_export>import. html but this is not working for me.
So can anyone know how to do that and explain me fully step by step?

My custom templates work perfectly for django's template like form and list, but for import-export didn't.

**Comment by matthewhegarty** [comment]
I suggest to search through issues and PRs to see if anyone has completed this previously.  For example, [this issue](https://github.com/django-import-export/django-import-export/issues/838).
